devimint/src/lib.rs

```diff
@@ -152,11 +152,14 @@ impl Gatewayd {
             }
         })
         .await?;
-        poll("gateway connect-fed", || async {
-            Ok(cmd!(self, "connect-fed", connect_str.clone())
-                .run()
-                .await
-                .is_ok())
+        poll_max_retries("gateway connect-fed", 10, || async {
```

Comment:
This will also prevent it from continue retrying for 10 minutes like it does in the run linked in the description.